addons/web/__manifest__.py

```diff
@@ -564,6 +564,7 @@
         'web._assets_backend_helpers': [
```

Comment:
Can't it be placed in `_assets_backend` ?
This bundle is used for other purposes

Reply:
@stefanorigano I moved it there : https://github.com/odoo-dev/odoo/pull/1365/files#diff-66d417d1828879bbc2124ddc118b47030ea7c30fb93e660c5231365420002214R194

Is it ok for you ?